node/src/main/scala/coop/rchain/node/api/WebApi.scala

```diff
@@ -214,7 +228,12 @@ object WebApi {
 
   final case class ApiStatus(
       version: Int,
-      message: String
+      versionInfo: String,
```

Comment:
This two versions seems confusing.

Let's create separate VersionInfo object which can have multiple fields for specific module version.

```scala
ApiStatus(version: VersionInfo, ...)
VersionInfo(api: String, node: String)
```
Maybe better also to have API version to be string if we want to add some minor versions in the future.